documentation/project-docs/developer-guide.md

```diff
@@ -37,12 +37,15 @@ The simple way to launch Visual Studio after building via `build.cmd` is to doub
 
 Alternatively, to open the solution in Visual Studio, be sure to build with `build.cmd` and run the generated environment for your shell. If you're using `cmd`, then run `artifacts\sdk-build-env.bat`. If you're using PowerShell, you need to 'dot source' `artifacts/sdk-build-env.ps1`. Finally, open Visual Studio with `devenv sdk.sln`.
 
-In addition, Visual Studio must have the following option set:
+In addition, Visual Studio must have the following option set (this option is automatically set in preview Visual Studio builds):
 
 ![image](https://user-images.githubusercontent.com/23152278/211684116-923ed37e-6d56-42bf-befe-a5ef66758000.png)
 
 Go to `Tools` -> `Options` to make sure "Use previews of the .NET SDK (requires restart)" is checked and restart VS.
 
+> [!NOTE]
+> If you're building main, we may be using a preview version of the SDK to build as specified in [global.json](../../global.json#L3). We only test preview SDKs with the latest Visual Studio previews so would recommend installing the latest preview build. You can see the preview versions we test with [here](https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#preview-versioning)
```

Comment:
I'm a little split on using ../../global.json#L3 as a link because that could easily change. We could use a permalink, but that would potentially link to an out-of-date document...

Less controversially, we should be able to cut en-us from the second link to make it language-agnostic.